environment/latent_code_receiver.py

```diff
@@ -0,0 +1,38 @@
+import socket
+import struct
+
+CHUNK_SIZE = 1024
+
+
+def receive_latent_codes(num_floats_per_latent_code: int) -> list:
+    """
+    Client receiving latent codes on localhost port 6969.
```

Comment:
Same here as with server-side (parameterize port etc). Once again, it is better to describe the functions action, i.e., "Receives latent codes using a socket client" or whatever

may also be worth noting that the latent codes are unwrapped into float array